server/real_estate/src/main/java/com/realestate/real_estate/repos/PropertyRepository.java

```diff
@@ -1,9 +1,21 @@
 package com.realestate.real_estate.repos;
 
+import java.util.List;
+import java.util.Optional;
+
 import org.bson.types.ObjectId;
 import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface PropertyRepository extends MongoRepository<Property, ObjectId> {
+public interface PropertyRepository extends MongoRepository<PropertyDetails, Integer> {
```

Comment:
Should be MongoRepository<PropertyDetails, ObjectId>

Reply:
Fixed it